src/tools/analyzer/modules/request.py

```diff
@@ -188,9 +196,14 @@ def print_formatted_verbose(self, source, patterns):
                 if plugin:
                     print("   - " + plugin)
                 if name:
-                    print("       - " + name[:-2])
+                    # Avoid duplicate CID output with the same CR #
+                    if cr not in self.done:
+                        print("       - " + name[:-2])
+                        cr_done.append(cr)
```

Comment:
How are `self.done` and `cr_done` related?

Reply:
self.done should have been removed and replaced by cr_done, I fixed this now and pushed to this PR.